web_generator/lib/src/interop_gen/transform/transformer.dart

```diff
@@ -227,19 +253,24 @@ class Transformer {
   }
 
   GenericType _transformTypeParamDeclaration(
-      TSTypeParameterDeclaration typeParam) {
+      TSTypeParameterDeclaration typeParam,
+      {bool shouldUseJSConstraint = true}) {
```

Comment:
I may be missing something, but this isn't set in any of the calls. Should we remove this parameter?